library/train_util.py

```diff
@@ -1071,14 +1075,18 @@ def __getitem__(self, index):
                 crop_ltrb = image_info.latents_crop_ltrb  # calc values later if flipped
                 if not flipped:
                     latents = image_info.latents
+                    mask = image_info.mask
                 else:
                     latents = image_info.latents_flipped
+                    mask = image_info.mask_flipped
 
                 image = None
             elif image_info.latents_npz is not None:  # FineTuningDatasetまたはcache_latents_to_disk=Trueの場合
                 latents, original_size, crop_ltrb, flipped_latents = load_latents_from_disk(image_info.latents_npz)
+                mask = load_mask(image_info.absolute_path, image_info.resized_size) / 225
                 if flipped:
                     latents = flipped_latents
+                    mask = mask.flip(mask, dims=[3])
```

Comment:
`mask` is an np.array here and doesn't have a `flip` method. `np.flip(mask, axis=1)` is probably what's desired here.